app/controllers/web_git/commands_controller.rb

```diff
@@ -4,8 +4,15 @@ class CommandsController < ::ApplicationController
 
     def status
       Dir.chdir(Rails.root) do
+        
+        unless Dir.exist?(".git")
+          puts "Initializing a git repository"
+          `git init`
+          `git add -A`
+          puts "Make first commit"
```

Comment:
Tiny nitpick: the tense of the verbs are changing in the output. Maybe "`Making first commit`"? Alternatively, "`Initialize a git repository`" above?